.travis.yml

```diff
@@ -96,6 +95,10 @@ jobs:
       if: branch = master AND type = push
       script:
         - yarn semantic-release
+        - npm config set '//npm.pkg.github.com/:_authToken' $GH_TOKEN
+        - npm config set registry https://npm.pkg.github.com/@typeform
+        - npm publish
```

Comment:
Does this publish the same code to both registries? `semantic-release` takes care of versioning and only releases when there are any changes (commit message starts with `fix`, `feat`, etc.). I think `npm publish` might work a bit differently.

Reply:
Nice catch, didn't think of that, yes, it'll try to publish with the version specified in `package.json`, and since `semantic-release` didn't make any changes there, it'll take current version. While here it's not that much of an issue since `package.json` has actual `version` (so it'll just fail because it won't be able to publish same version), in other packages it will publish whatever is specified in `package.json#version` 🤔

Reply:
💩

Reply:
We do not update the version in `package.json`, in most projects it has a value of `"version": "0.0.0-semantic-release",` and we let semantic release take care of it.

I see the version is changed here by @prilutskiy but he was resolving some issues with `semantic-release` so this might be just a temporary thing. Am I correct?

Reply:
okay, yeah, I see that you don't commit `package.json` after release, I only saw changed version in `package.json`. I can still think of another safe solution: use plugin with shortly config:
`["@sematic-release/exec", { "successCmd": "npm publish" }]`

Reply:
Previously we were using outdated version of semantic release. Max should be having a look to let us know if your approach works.

Reply:
A couple of notes:
1. Version in package.json is now handled automagically, by semantic release
2. To publish the same thing to an additional channel, I recommend using semantic-release also. This is how its config looks like currently:

```
{
  "branches": [
    "master"
  ],
  "plugins": [
    "@semantic-release/commit-analyzer",
    "@semantic-release/release-notes-generator",
    "@semantic-release/changelog",
    "@semantic-release/npm",
    "@semantic-release/git",
    "@semantic-release/github"
  ]
}
```

In my opinion, to achieve the goal it makes sense to add something like `@semantic-release/github-package-registry`, in addition to the npm plugin specified here. I'm not sure such plugin exists already, but in case it does not - there's a thing called [@semantic-release/exec](https://github.com/semantic-release/exec)

Reply:
UPD:

Found [something](https://github.com/gr2m/semantic-release-github-npm-registry), but maybe we'd just go with our custom approach, if the tool is not that good

Reply:
@prilutskiy I think there is no proper solution that does what we need
I'm using just `npm publish` because I want to rely on `@semantic-release/npm` complete everything for me (e.g. if `npm pack` is needed to generate `tar` with updated `version` and `dist` fields updated in `package.json`, then just run [`npm publish`](https://github.com/semantic-release/npm/blob/680a82774118ecd935e07c2d480e1f5289c07180/lib/publish.js#L23)